licenses (#25181)
docker/docker.github.io: content/manuals/subscription/scale.md

- Testcontainers Cloud runtime minutes
- Docker Hardened Images (DHI) repositories

## Add licenses

Licenses add a layer to your Docker subscriptions, letting you assign products to select members of your
organization. You can purchase licenses for some Docker products:

- AI Governance
- Docker Offload 

Once you've purchased licenses for your organization, you can [manage license assignment](/manuals/admin/organization/manage/manage-licenses.md) from the **Members** page in Docker Home.

> [!TIP]
> To purchase licenses for AI Governance and Docker Offload, [contact sales](https://www.docker.com/pricing/contact-sales/). 

> ### AI Governance licenses

Reviewer: If licenses for ai governance aren't tied to subscriptions, why does this content live here? (I also can't think of a better place to put this, but something we should consider post-release and perhaps with your subscription freshness work)

Reviewer: Perhaps we actually need a whole new top-level item under 'Platform' called 'licenses'? (Or perhaps not depending on the outcome of Tuesday's meeting with Greg and Patrick)

AI Governance licenses let admins create and apply organization-wide AI Governance policies for license-holding members.

- AI Governance licenses are separate from Docker Team or Business. 

> - You can purchase AI Governance licenses without an existing subscription.

Reviewer: This is almost the same point as above, I'd merge the two

- When you purchase AI Governance licenses, you may assign them to organization members without a Docker subscription seat. 
- AI Governance licenses currently applies to Docker Sandbox usage.

> When a member who holds an AI governance license uses AI products like Docker Sandbox, the organization policy overrides the member’s local policy rules. Members without AI Governance licenses can still use AI products. 

Reviewer 2: ```suggestion
When a member who holds an AI governance license uses Docker Sandbox, the organization policy overrides the member’s local policy rules. Members without AI Governance licenses can still use Docker Sandbox, but organization policies will not govern their usage. 
```

### Docker Offload licenses 

Docker Offload lets developers offload building and running containers to the cloud. 

- You must have a Docker Team or Docker Business subscription.
- You can only assign Docker Offload licenses to members with Docker Team or Docker Business.

## Add minutes

Minutes don't roll over. Base subscription minutes reset each billing period and don't accumulate. Additional purchased minutes expire at the end of your subscription period.

For example, with an annual Docker Team subscription (500 included minutes), if you purchase 500 additional minutes, only the additional 500 minutes roll over until your annual renewal.

> ### Docker Build Cloud build minutes

Author: @aevesdocker How would I handle redirects for this? 🤔

> ### Docker Build Cloud build minutes

Author: Oh, I'm sorry Docker Agent. I didnt know you were also known as aevesdocker...

> ### Docker Build Cloud build minutes

Reviewer: I'd ignore, this has never been an issue before

> ### Docker Build Cloud build minutes

Reviewer 3: I am also not aevesdocker but I'd say let's not worry about broken deep/anchor links. Stick to page-level redirects.
